kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/util/TextFieldValidator.java

```diff
@@ -76,7 +76,7 @@ public void onChange(ChangeEvent event) {
         NETWORK("network", "(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})/(\\d{1,3})"),
         IPv4_ADDRESS("ipv4_address", "\\b(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\b"),
         PORT("port", "^[0-9]*$"),
-        PORT_RANGE("port_range", "^[0-9]+:*[0-9]+$"),
+        PORT_RANGE("port_range", "^[0-9]+:[0-9]+$"),
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"